### PR TITLE
feat(operator): add-pressure-exec replace job(get pod log)

### PR DESCRIPTION
**Pull request by moomman**
### Type of change:

- [x] Bugfix
- [x] New feature provided
- [ ] Improve performance

### What this PR does / why we need it:
1. add pressureCfg spec for pressure-exec
2. change status Msg to express steady And chaos experiment 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
#272

### CODE REVIEW

**Comment by Xu-Wentao** [comment]
add more meaningful test to cover your core func like `exec pressure`

File: shardingsphere-operator/pkg/pressure/pressure_test.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/apache/shardingsphere-on-cloud/shardingsphere-operator/api/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"regexp"
+	"testing"
+	"time"
+)
+
+func TestRun(t *testing.T) {
```
Comment (Xu-Wentao) [inline shardingsphere-operator/pkg/pressure/pressure_test.go:33]: use `ginkgo` and `omega` replace of `testing` and `assert`

File: shardingsphere-operator/pkg/pressure/utils.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	"math/rand"
+	"strings"
+	"time"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
+
+const charset = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```
Comment (Xu-Wentao) [inline shardingsphere-operator/pkg/pressure/utils.go:31]: `golang-sdk` has `random string` func.
https://github.com/database-mesh/golang-sdk/tree/main/pkg/random

**Review by Xu-Wentao — APPROVED** [review]
lgtm

**Review by mlycore — APPROVED** [review]

**Pull request merged**